packages/bits/src/lib/table/table-cell/table-header-cell.component.ts

```diff
@@ -86,6 +86,9 @@ export class TableHeaderCellComponent extends CdkHeaderCell implements OnInit, O
         return this.sortingState.isColumnSorted;
     }
 
+    @HostBinding(`class.${FIXED_WIDTH_CLASS}`)
```

Comment:
I just noticed we already have a binding for this at line 64. In that case, `widthFixed` can be just a simple property, and the condition in the existing binding can be changed to this:

```
return this.isIconCell || this.fixedWidth;
```

Reply:
I cant just use the get one because It also needs to be set. see how i changed it.